README.rst

```diff
@@ -78,3 +82,6 @@ Configure your webserver:
 
 As final step you need to point your external webserver to your xmpp-http-upload flask app.
 Check the ``contrib`` directory, there is an example for nginx there.
+=======
```

Comment:
That merge somehow went wrong. This line should be blank and the items belong to the list at the end of the previous section.

(Don’t worry if you don’t want to, I’ll clean this up when I merge the PR otherwise.)